templates/people_admin/people_matcher.html

```diff
@@ -2,70 +2,124 @@
 {% load static %}
 
 {% block scripts %}
-  <script src="{% static "bundles/dashboards.js" %}"></script>
+<script src="{% static " bundles/dashboards.js" %}"></script>
+<script>
+  // get cookie for form POST
+  function getCookie(name) {
```

Comment:
sorry, should have mentioned we have a library for this: (see js/follow-button for example)

Reply:
I did try to use it but got a `Cannot use import statement outside a module` error so just thought I'd run with it. Was definitely a thing that made me wish I did it in react.